README.md

```diff
@@ -10,15 +10,15 @@
 
 **Open source, cloud-native security and policy project**
 
-`cnspec` is a cloud-native solution to assess the security and compliance of your business-critical infrastructure. `cnspec` finds vulnerabilities and misconfigurations on all systems in your infrastructure including: public and private cloud environments, Kubernetes clusters, containers, container registries, servers, and endpoints, SaaS products, infrastructure as code, APIs, and more.
+cnspec assesses your entire infrastructure's security and compliance. It finds vulnerabilities and misconfigurations on all your systems, including: public and private cloud environments, Kubernetes clusters, containers, container registries, servers, and endpoints, SaaS products, infrastructure as code, APIs, and more.
```

Comment:
systems isn't the right word since we're covering SaaS as well

Reply:
```suggestion
cnspec assesses your entire infrastructure's security and compliance. It finds vulnerabilities and misconfigurations throughout public and private cloud environments, Kubernetes clusters, containers, container registries, servers, and endpoints, SaaS products, infrastructure as code, APIs, and more.
```